app/models/partner.rb

```diff
@@ -507,13 +507,13 @@ def three_or_less_category_tags
     errors.add :categories, 'Partners can have a maximum of 3 Category tags'
   end
 
-  def partnership_admins_must_add_tag
+  def partnership_admins_must_add_partnership
     return if accessed_by_user.nil? # HACK: to stop factory breaking tests
     return unless accessed_by_user.partnership_admin?
 
-    if tags.any?
-      accessed_by_user.tags.each do |t|
-        return true if tags.include? t
+    if partnership_ids.any?
+      accessed_by_user.tags.pluck(:id).each do |t|
```

Comment:
So on the user still has `tags` which may match a `partnership.id` not `partnerships` like partner. Do we want to keep it like this or is it confusing?

Reply:
Is there any danger it matches the id of another type of tag? Just riffing here - haven't dug into the code change from when we split the partner tags. I', assuming all other entities kept the generic tag relation instead.

Reply:
I think this can poss be refactored to remove reference to `tags` yes but for now it works and we already have a ticket open to resolve removing all refs to `tags` :)